CreatingPackage.md

```diff
@@ -0,0 +1,124 @@
+# CI / CD
+We utilize GitHubActions, CodeCov, and Documenter.jl to automate testing, code coverage reporting, and documentation deployment.
+## Automatic Documentation with Documenter.jl
+### Setup
+If your package was set up using `PkgTemplates.jl`, no changes to the workflow file are necessary. Otherwise, follow the provided reference file.
+
+Generate a `Deploy Key` and a `Documenter-Key` by using the `DocumenterTools` Julia package. Execute 
+```julia 
+DocumenterTools.genkeys(user="Username", repo="MyPackage.jl")
+```
+ Ensure accurate capitalization for the username and repository. Follow the provided instructions for adding the keys to GitHub, granting write access to the `Deploy Key`.
+
+If this has been set up correctly, the next time a significant change is being pushed to your repository (according to your workflow-triggers), GitHub Actions should now additionally build the documentation in a new branch called `gh-pages`. 
+
+Please refer to the [Documenter.jl](https://documenter.juliadocs.org/stable/man/hosting/) documentation for Hosting.
+### GitHub Pages (`gh-pages`)
+Set up automatic deployment on GitHub Pages as follows:
+- Navigate to the “Code and Automation” > “Pages” section within your repository settings. Find the section "Build and deployment"
+- Confirm that the “Source” is set to `Deploy from branch` and the “Branch” is `gh-pages` `/(root)`.
+
+Keep in mind that documentation updates may take a few minutes to become visible as the workflow has to run.
+
+Please refer to the [gh-pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages) documentation.
+## CodeCov
+To integrate CodeCov:
+- Register or log in via GitHub at the [CodeCov website](https://about.codecov.io).
+- Locate your package on the Repos-Overview page and select `Setup repo`.
+- Follow the provided instructions to complete the setup. Note that "Step 2: add Codecov to your [GitHub Actions workflow yaml file](https://github.com/JeanAnNess/Covid_tracker/tree/main/.github/workflows)" is done by the specifying the `Codecov` plugin while creating your Template via `PkgTemplates`.
+## Troubleshooting and Tips
+
+If the documentation for a function is not appearing when using 
+```julia-repl
+?help function
+```
+- Ensure that you have executed `using MyPackage`.
+- Verify that the docstring is placed directly above the function definition with no intervening spaces.
+
+For more detailed guidance, refer to the relevant documentation for Documenter.jl, GitHub Actions, and gh-pages.
+# References
+- [julia](https://julialang.org)
+- [git](https://git-scm.com/)
+- [GitHub](https://github.com)
+- [gh-pages](https://docs.github.com/en/pages/quickstart)
```

Comment:
A bullet point that says "gh-pages" might not be informative enough for students.

Reply:
This has been changed to "GitHub Pages" to align with the "GitHub Actions" reference